src/text-tags.js

```diff
@@ -42,6 +42,22 @@ function buildTagsFromStory(config, story, url = {}) {
   return storyMetaData;
 }
 
+function buildTagsFromTopic(config, data, url = {}) {
+
+  const topicUrl = `${config['sketches-host']}/topic/${data.tag}`;
+
+  const topicMetaData = {
+    "page-title":data.tagName,
+    description: data.tagDescription || data.tagName,
```

Comment:
like section we might have to fallback to home data if tag related data is not there.

Reply:
done...

Reply:
done...

Reply:
done...